nekoyume/Assets/_Scripts/UI/Widget/Menu.cs

```diff
@@ -728,56 +729,101 @@ protected override void Update()
         {
             base.Update();
 
-            if (Input.GetKey(KeyCode.LeftControl))
+            if (!Input.GetKey(KeyCode.LeftControl))
             {
-                if (Input.GetKeyDown(KeyCode.C) &&
-                    !Find<CombinationResultPopup>().gameObject.activeSelf)
-                {
-                    Find<CombinationResultPopup>().ShowWithEditorProperty();
-                }
-                else if (Input.GetKeyDown(KeyCode.E) &&
-                         !Find<EnhancementResultPopup>().gameObject.activeSelf)
+                return;
+            }
+
+            if (Input.GetKeyDown(KeyCode.C))
+            {
+                var game = Game.Game.instance;
+                var states = game.States;
+                var avatarAddr = states.CurrentAvatarState.address;
+                ActionManager.Instance.ClaimItems(
+                    (
+                        avatarAddr,
+                        new[]
+                        {
+                            FungibleAssetValue.FromRawValue(
+                                Currency.Uncapped(
+                                    ticker: "Item_T_400000",
+                                    decimalPlaces: 0,
+                                    minters: null),
+                                100)
+                        }));
+            }
+
+            if (Input.GetKey(KeyCode.T))
+            {
+                if (Input.GetKeyDown(KeyCode.Alpha1))
                 {
-                    Find<EnhancementResultPopup>().ShowWithEditorProperty();
+                    var game = Game.Game.instance;
+                    var states = game.States;
+                    var sender = states.AgentState.address;
+                    ActionManager.Instance.TransferAsset(
+                        sender,
+                        new Address("0x9723f230A20372564c50d63403223e83549a477f"),
```

Comment:
What is this code for testing?

Reply:
🤔 Yes it just for testing, And.. It's better to delete. I’ll open a pull request. Thanks!

Reply:
https://github.com/planetarium/NineChronicles/pull/3155